api/task.go

```diff
@@ -68,9 +68,19 @@ const (
 type TaskDatabasePITRRestorePayload struct {
 	// The project owning the database.
 	ProjectID int `json:"projectId,omitempty"`
+
+	// TargetInstance must be within the same environment as the instance of the original database.
+	// Only used when doing PITR to a new database.
+	TargetInstance *Instance `json:"targetInstance,omitempty"`
```

Comment:
Shall we use TargetInstanceID instead, similar to ProjectID convention above? We don't want to persist the whole Project entity structure in the payload.

Reply:
SGTM! : )